File: src/main/java/com/allure/generator/config/JacksonConfig.java
package com.allure.generator.config;

import org.babyfish.jimmer.jackson.ImmutableModule;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class JacksonConfig {
    @Bean
    public ImmutableModule immutableModule() {
        return new ImmutableModule();
    }
}

File: src/main/kotlin/com/mellivorines/generator/config/JacksonConfig.kt
package com.mellivorines.generator.config

import org.babyfish.jimmer.jackson.ImmutableModule
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class JacksonConfig {
    @Bean
    fun immutableModule(): ImmutableModule = ImmutableModule()
}